8.frontend-react/crm-frontend/src/components/lineChart/LineChart.jsx

```diff
@@ -0,0 +1,56 @@
+import React, {useState, useEffect} from 'react';
+import { Line } from 'react-chartjs-2';
+import './lineChart.scss';
+import ColorsHelper from '../../helpers/colorsHelper';
+
+function LineChart(props){
+  const colors = new ColorsHelper();
+  const [data, setData] = useState({labels: [], datasets:[]});
+
+  useEffect(()=>{
+    if(props.data){
+      const datasets = [];
```

Comment:
extract the content here into a function. 
you are not suppose to have a big bunch of code inside an effect function